### ISSUE TITLE
Docker container errors

### ISSUE THREAD

**Issue by psybernoid** [Control-D-Inc/ctrld#80]
When launching a container on a Rasp Pi 3.

When launched, the logs show:

```
Aug 30 05:56:48.000 ERR could not listen and serve on: 172.21.0.2:53 error="listen udp 172.21.0.2:53: bind: address already in use"
Aug 30 05:56:48.000 WRN could not listen on udp: 172.21.0.2:53 error="listen udp 172.21.0.2:53: bind: address already in use"
Aug 30 05:56:48.000 ERR could not listen and serve on: 172.21.0.2:53 error="listen tcp 172.21.0.2:53: bind: address already in use"
Aug 30 05:56:48.000 WRN could not listen on tcp: 172.21.0.2:53 error="listen tcp 172.21.0.2:53: bind: address already in use"
```

Before launching, there's nothing listening on port 53:

```
Active Internet connections (only servers)
Proto Recv-Q Send-Q Local Address           Foreign Address         State       PID/Program name
tcp        0      0 0.0.0.0:22              0.0.0.0:*               LISTEN      403/sshd: /usr/sbin
tcp6       0      0 :::22                   :::*                    LISTEN      403/sshd: /usr/sbin
udp        0      0 0.0.0.0:68              0.0.0.0:*                           308/dhclient
```

After launching, netstat returns:
``` 
Active Internet connections (only servers)
Proto Recv-Q Send-Q Local Address           Foreign Address         State       PID/Program name
tcp        0      0 0.0.0.0:53              0.0.0.0:*               LISTEN      2132/docker-proxy
tcp        0      0 0.0.0.0:22              0.0.0.0:*               LISTEN      403/sshd: /usr/sbin
tcp6       0      0 :::53                   :::*                    LISTEN      2140/docker-proxy
tcp6       0      0 :::22                   :::*                    LISTEN      403/sshd: /usr/sbin
udp        0      0 0.0.0.0:53              0.0.0.0:*                           2154/docker-proxy
udp        0      0 0.0.0.0:68              0.0.0.0:*                           308/dhclient
udp6       0      0 :::53                   :::*                                2160/docker-proxy
```

nslookup from another machine does provide results:

```
> yahoo.com
Server:  [172.25.30.253]
Address:  172.25.30.253

Non-authoritative answer:
Name:    yahoo.com
Addresses:  2001:4998:124:1507::f000
          2001:4998:24:120d::1:1
          2001:4998:24:120d::1:0
          2001:4998:44:3507::8000
          2001:4998:44:3507::8001
          2001:4998:124:1507::f001
          54.161.105.65
          74.6.231.20
          74.6.143.25
          74.6.231.21
          34.225.127.72
          74.6.143.26
          98.137.11.163
          98.137.11.164
```

However, as soon as the ctrld container is queried, the logs fill up with this message:

```
Aug 30 05:57:48.000 ERR could not lookup IP error="read udp 172.21.0.2:43366->172.21.0.1:53: i/o timeout"
Aug 30 05:57:48.000 ERR could not lookup IP error="read udp 172.21.0.2:54253->172.21.0.1:53: i/o timeout"
Aug 30 05:57:48.000 ERR could not lookup IP error="read udp 172.21.0.2:39888->172.21.0.1:53: i/o timeout"
Aug 30 05:57:48.000 ERR could not lookup IP error="read udp 172.21.0.2:53613->172.21.0.1:53: i/o timeout"
Aug 30 05:57:48.000 ERR could not lookup IP error="read udp 172.21.0.2:51786->172.21.0.1:53: i/o timeout"
Aug 30 05:57:48.000 ERR could not lookup IP error="read udp 172.21.0.2:59539->172.21.0.1:53: i/o timeout"
Aug 30 05:57:48.000 ERR could not lookup IP error="read udp 172.21.0.2:51193->172.21.0.1:53: i/o timeout"
Aug 30 05:57:48.000 ERR could not lookup IP error="read udp 172.21.0.2:45256->172.21.0.1:53: i/o timeout"
Aug 30 05:57:48.000 ERR could not lookup IP error="read udp 172.21.0.2:55510->172.21.0.1:53: i/o timeout"
Aug 30 05:57:48.000 ERR could not lookup IP error="read udp 172.21.0.2:36988->172.21.0.1:53: i/o timeout"
Aug 30 05:57:48.000 ERR could not lookup IP error="read udp 172.21.0.2:36032->172.21.0.1:53: i/o timeout"
Aug 30 05:57:48.000 ERR could not lookup IP error="read udp 172.21.0.2:34299->172.21.0.1:53: i/o timeout"
Aug 30 05:57:48.000 ERR could not lookup IP error="read udp 172.21.0.2:38708->172.21.0.1:53: i/o timeout"
Aug 30 05:57:48.000 ERR could not lookup IP error="read udp 172.21.0.2:53415->172.21.0.1:53: i/o timeout"
Aug 30 05:57:48.000 ERR could not lookup IP error="read udp 172.21.0.2:46353->172.21.0.1:53: i/o timeout"
Aug 30 05:57:48.000 ERR could not lookup IP error="read udp 172.21.0.2:60513->172.21.0.1:53: i/o timeout"
Aug 30 05:57:48.000 ERR could not lookup IP error="read udp 172.21.0.2:44992->172.21.0.1:53: i/o timeout"
Aug 30 05:57:48.000 ERR could not lookup IP error="read udp 172.21.0.2:47081->172.21.0.1:53: i/o timeout"
Aug 30 05:57:48.000 ERR could not lookup IP error="read udp 172.21.0.2:49327->172.21.0.1:53: i/o timeout"
Aug 30 05:57:48.000 ERR could not lookup IP error="read udp 172.21.0.2:54442->172.21.0.1:53: i/o timeout"
Aug 30 05:57:48.000 ERR could not lookup IP error="read udp 172.21.0.2:45936->172.21.0.1:53: i/o timeout"
Aug 30 05:57:48.000 ERR could not lookup IP error="read udp 172.21.0.2:34393->172.21.0.1:53: i/o timeout"
Aug 30 05:57:48.000 ERR could not lookup IP error="read udp 172.21.0.2:36370->172.21.0.1:53: i/o timeout"
Aug 30 05:57:48.000 ERR could not lookup IP error="read udp 172.21.0.2:37519->172.21.0.1:53: i/o timeout"
Aug 30 05:57:48.000 ERR could not lookup IP error="read udp 172.21.0.2:60778->172.21.0.1:53: i/o timeout"
Aug 30 05:57:48.000 ERR could not lookup IP error="read udp 172.21.0.2:37952->172.21.0.1:53: i/o timeout"
Aug 30 05:57:48.000 ERR could not lookup IP error="read udp 172.21.0.2:47944->172.21.0.1:53: i/o timeout"
```

**Comment by cuonglm** [comment]
This is the error message from ptr lookup discovery. Next release will lower the message from error to warn level.

In the meantime, you can set `discover_ptr = false` in your config to disable ptr discover.

**Comment by cuonglm** [comment]
> @cuonglm May be good to make error more explicit too: "could not perform PTR lookup"

Sure, sounds definitely better.

**Comment by cuonglm** [comment]
This issue is fixed in v1.3.1 release.
